src/materialized/tests/sql.rs

```diff
@@ -544,3 +546,99 @@ fn test_temporary_views() -> Result<(), Box<dyn Error>> {
 
     Ok(())
 }
+
+// This test attempts to observe a linearizability violation by creating a set of
+// sources which are constantly being appended to, then creating a materialized
+// view of each of their sizes, then repeatedly reading from some subset of
+// those materialized views. If any of the sizes ever decrease from one read to
+// the next, linearizability has been violated.
+//
+// The sources are based off of n CSVs, each named in<i>.csv, a source for each
+// one named s<i> which tails the CSV, and a materialized view for each named
+// v<i>.
+//
+// N.B. this test currently fails and is ignored. TODO(justin): fix it.
+// N.B. this test also fails more reliably in release mode.
+#[test]
+#[ignore]
+fn test_linearizable() -> Result<(), Box<dyn Error>> {
+    ore::test::init_logging();
+    let config = util::Config::default();
```

Comment:
Am I crazy or does this `config` not get used?

Reply:
Oh whoops, you're right, I guess the compaction window isn't needed anyway.